tests/unit/test_keycloak.py

```diff
@@ -0,0 +1,107 @@
+# *****************************************************************************
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+# ******************************************************************************
+import unittest
+
+from dlab_core.domain.logger import INFO
+from dlab_core.infrastructure.logger import StreamLogBuilder
+from dlab_core.infrastructure.services import (
+    KeyCloak, InvalidParameterFormatError
+)
+
+from mock import patch
+
+from requests.exceptions import RequestException
+
+MOCK_PUBLIC_KEY = 'secret'
+MOCK_VALID_TOKEN = (
+    'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJuYW1lIjoiWXVyYSJ9.bG1BjdKLzqdx1qho'
+    'eg4B598nV9JOds0-23xgRXG-n5c'
+)
+MOCK_INVALID_TOKEN = (
+    'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJuYW1lIjoiWXVyYSJ9.bG1BjdKLzqdx1qho'
+)
+
+
+class MockResponse:
+    def __init__(self, json_data=None, ok=True):
+        self.json_data = {
```

Comment:
you have two states of active maybe better will be to send active in args then json?